src/vision/src/HSVFilterNode.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: November 11th, 2017
+ * Description: Performs an HSV filter on incoming data and republishes it
+ */
+
+#include <HSVFilterNode.h>
+
+HSVFilterNode::HSVFilterNode(int argc, char** argv, std::string node_name) {
+    // Setup NodeHandles
+    ros::init(argc, argv, node_name);
+    ros::NodeHandle nh;
+    ros::NodeHandle private_nh("~");
+    image_transport::ImageTransport it(nh);
+
+    std::string subscribeTopic = "/camera/image_raw";
+    std::string publishTopic   = "/vision/output";
+
+    XmlRpc::XmlRpcValue hsv;
+    if (!nh.getParam("hsv", hsv)) {
```

Comment:
We should be using `private_nh` here instead of `nh`, so that we're looking up params only under this nodes param namespace. Avoiding doing things in global scope helps prevent conflicts.

Reply:
done